CONTRIBUTING.md

```diff
@@ -81,6 +81,15 @@ Our merge strategy relies on the classification of PRs on two axes:
 * How controversial are the design decisions
 * How complex is the implementation
 
+Each issue [label](https://github.com/bevyengine/bevy/labels) has a prefix denoting its category:
+
+* A: Area
+* C: Category
+* D: Difficulty
+* O: Operating system
+* P: Priority
+* S: Status
```

Comment:
Maybe add an explanation for each category? When I first contributed to Bevy I didn't understand what's the difference between A and C.

Reply:
Maybe add some example labels

Reply:
Example labels are a good way to do this.